### ISSUE TITLE
cert-manager has deprecated v1alpha3 API in v1.4+, disabled it in v1.6+ and removed it in v1.7+

### ISSUE THREAD

**Issue by waxling** [kmorning/cert-manager-webhook-desec#1]
As per https://cert-manager.io/docs/installation/upgrading/remove-deprecated-apis/

the v1alpha3 api is deprecated in 1.4, disabled in 1.6 and removed in 1.7 of cert-manager. 

In addition it looks like there migh be a number of updates that could be merged-in. Is this being actively maintained? Happy to help out (although my GO clue is low). 

e.g. I could submit a PR for the api change :
sed -i 's/v1alpha3/v1/g' examples/deploy/desec-webhook.yaml

looks like it would cover the change.

**Comment by kmorning** [comment]
Hi waxling, sorry for the late reply.  As far as actively maintained goes, I originally put this together to solve a task at hand and have basically left it alone since.   I had intended to look at this again as time allowed, but it was probably going to take something breaking in my particular use case for me to revisit this.  I haven't updated cert-manager in my cluster since I did this so I'm still sitting at v1.1.0.  So yes, I'm overdue on this.
With that said, I certainly won't turn down any help offered and it would be much appreciated.   To be honest, I haven't done any real collaborative development on github before.   If you'd like to submit the PR for this issue, that would be great as I'd like to go through the process.
